arcade/gui/widgets/text.py

```diff
@@ -488,8 +487,10 @@ def __init__(
 
         # Set how fast the mouse scroll wheel will scroll text in the pane.
         # Measured in pixels per 'click'
-        self.scroll_speed = scroll_speed if scroll_speed is not None \
-                            else font_size
+        self.scroll_speed = (
+            scroll_speed if scroll_speed is not None
+            else font_size
+        )
```

Comment:
Is this related to pyright?  It's fine if not, the code looks better when parens.  Just wondering.

Reply:
The line was too long and had originally used a slash which I find hard to read. If you have a recommendation on how to format it I'm happy to update it

Reply:
No recommendation, I wanted to confirm it wasn't pyright choking on backslashes or something.  I agree with your formatting.